src/modules/repository/implementation/sequelize/migrations/20241215122200-create-paranet-assets.js

```diff
@@ -0,0 +1,60 @@
+export async function up({ context: { queryInterface, Sequelize } }) {
+    await queryInterface.createTable('paranet_assets', {
+        id: {
+            type: Sequelize.INTEGER,
+            primaryKey: true,
+            autoIncrement: true,
+        },
+        blockchain_id: {
+            type: Sequelize.STRING,
+            allowNull: false,
+        },
+        ual: {
+            type: Sequelize.STRING,
+            allowNull: false,
+        },
+        paranet_ual: {
+            type: Sequelize.STRING,
+            allowNull: false,
+        },
+        public_assertion_id: {
+            type: Sequelize.STRING,
+            allowNull: true,
+        },
+        private_assertion_id: {
+            type: Sequelize.STRING,
+            allowNull: true,
+        },
+        sender: {
+            type: Sequelize.STRING,
+            allowNull: true,
+        },
+        transaction_hash: {
+            type: Sequelize.STRING,
+            allowNull: true,
+        },
+        error_message: {
+            type: Sequelize.TEXT,
+            allowNull: true,
+        },
+        is_synced: {
+            type: Sequelize.BOOLEAN,
+            allowNull: false,
+            defaultValue: false,
+        },
+        created_at: {
+            allowNull: false,
+            type: Sequelize.DATE,
+            defaultValue: Sequelize.literal('NOW()'),
+        },
+        updated_at: {
```

Comment:
There needs to be a trigger for this, in raw SQL it is created like this:

```
        CREATE TRIGGER before_update_paranet_asset
        BEFORE UPDATE ON paranet_asset
        FOR EACH ROW
        SET NEW.updated_at = NOW();
```

Reply:
We should also add indexes on columns we do queries on:

Used to find missed assets
```
        CREATE INDEX idx_paranet_asset_updated_at_retries
        ON paranet_asset(updated_at, retries, isSynced);
```
Used to find all paranet assets
```
        CREATE INDEX idx_paranet_asset_paranet_ual_ual
        ON paranet_asset(ual, paranet_ual);
```
Not sure if we need any other indexes atm

Reply:
```
CREATE INDEX idx_paranet_asset_paranet_ual_syneced
        ON paranet_asset(synced, paranet_ual);
```

Reply:
no need for first trigger, its done automatically by sequelize 99%, but will test it when i can